mon-pix/app/components/challenge-statement.js

```diff
@@ -33,25 +36,25 @@ export default Component.extend({
       };
     }
     return undefined;
-  }),
+  }
 
   init() {
-    this._super(...arguments);
+    super.init(...arguments);
     this.id = 'challenge_statement_' + this.get('challenge.id');
-  },
+  }
 
-  attachmentsData: computed('challenge.attachments', function() {
+  @computed('challenge.attachments')
+  get attachmentsData() {
     return this.get('challenge.attachments');
-  }),
+  }
 
-  actions: {
-    selectAttachementUrl(attachementUrl) {
-      this.set('selectedAttachmentUrl', attachementUrl);
-    }
-  },
+  @action
+  selectAttachementUrl(attachementUrl) {
+    this.set('selectedAttachmentUrl', attachementUrl);
```

Comment:
```suggestion
    this.selectedAttachmentUrl = attachementUrl;
```
Je vais arrêter avec ces suggestions pour la suite. Si t'as décidé de passer ça sur cette PR je t'aide là-dessus. Autrement marque en résolu les commentaires de ce style. Sorry du spam

Reply:
C'est prévu dans une autre PR dédiée à cet effet.
Je préfère faire l'intentions la plus simple possible pour capitaliser au plus vite dessus

Reply:
:+1: